pkg/services/object/get/service.go

```diff
@@ -2,19 +2,39 @@ package getsvc
 
 import (
 	"github.com/nspcc-dev/neofs-node/pkg/core/client"
-	"github.com/nspcc-dev/neofs-node/pkg/core/netmap"
 	"github.com/nspcc-dev/neofs-node/pkg/local_object_storage/engine"
 	"github.com/nspcc-dev/neofs-node/pkg/services/object/util"
-	"github.com/nspcc-dev/neofs-node/pkg/services/object_manager/placement"
-	cid "github.com/nspcc-dev/neofs-sdk-go/container/id"
+	netmapsdk "github.com/nspcc-dev/neofs-sdk-go/netmap"
 	"github.com/nspcc-dev/neofs-sdk-go/object"
 	oid "github.com/nspcc-dev/neofs-sdk-go/object/id"
 	"go.uber.org/zap"
 )
 
+// NeoFS provides access to the NeoFS network to get information necessary for
+// the [Service] to work.
+type NeoFS interface {
```

Comment:
i was not agree last time about such naming so i have to write it one more time: i do not understand this interface, everything in this repo is about NeoFS. i would have understood if it was about some NeoFS constants or the things that are fixed and cannot be explained in NeoFS, but the methods here are about Network Map and that is it.
it really confuses me when i see this:
```go
type Service struct {
	*cfg
	neoFS NeoFS
}
```

Reply:
not a problem, can be renamed to any. `NeoFSNetwork` / `NeoFSChain`?

Reply:
`NeoFSNetwork` is ok to me

Reply:
`Netmapper`.